### PR TITLE
Fix/2229 french translation for share content and upload permission

### PR DESCRIPTION
**Pull request by inkhey**
related to #2229

## Checkpoints

The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

### CODE REVIEW

File: backend/development.ini.sample [hunk 1]
```diff
@@ -289,7 +289,7 @@ email.notification.created_account.subject = [{website_title}] Someone created a
 email.notification.share_content_to_emitter.subject = [{website_title}] you shared {content_filename} to {nb_receivers} users
 email.notification.share_content_to_receiver.subject = [{website_title}] {emitter_name} shared content {content_filename} with you
 email.notification.upload_permission_to_emitter.subject = [{website_title}] you give upload permission on {workspace_name} to {nb_receivers} users
-email.notification.upload_permission_to_receiver.subject = [{website_title}] {emitter_name} give you access to upload content into {workspace_name} share space
+email.notification.upload_permission_to_receiver.subject = [{website_title}] {emitter_name} give you access to upload content into share space {workspace_number}
```
Comment (PhilippeAccorsi) [inline backend/development.ini.sample]: replace `give you access to upload content into share space` by `allowed you to upload files in this shared space "{workspace_number}"`

File: backend/development.ini.sample [hunk 2]
```diff
@@ -289,7 +289,7 @@ email.notification.created_account.subject = [{website_title}] Someone created a
 email.notification.share_content_to_emitter.subject = [{website_title}] you shared {content_filename} to {nb_receivers} users
 email.notification.share_content_to_receiver.subject = [{website_title}] {emitter_name} shared content {content_filename} with you
 email.notification.upload_permission_to_emitter.subject = [{website_title}] you give upload permission on {workspace_name} to {nb_receivers} users
```
Comment (PhilippeAccorsi) [inline backend/development.ini.sample]: replace `you give upload permission on {workspace_name} to {nb_receivers} users` by `You allowed {nb_receivers} users to upload files on "{workspace_name}"`

File: backend/development.ini.sample [hunk 3]
```diff
@@ -289,7 +289,7 @@ email.notification.created_account.subject = [{website_title}] Someone created a
 email.notification.share_content_to_emitter.subject = [{website_title}] you shared {content_filename} to {nb_receivers} users
 email.notification.share_content_to_receiver.subject = [{website_title}] {emitter_name} shared content {content_filename} with you
```
Comment (PhilippeAccorsi) [inline backend/development.ini.sample]: replace `shared content {content_filename} with you` by `shared the file "{content_filename}" with you`

File: backend/development.ini.sample [hunk 4]
```diff
@@ -289,7 +289,7 @@ email.notification.created_account.subject = [{website_title}] Someone created a
 email.notification.share_content_to_emitter.subject = [{website_title}] you shared {content_filename} to {nb_receivers} users
```
Comment (PhilippeAccorsi) [inline backend/development.ini.sample]: Replace `you shared {content_filename} to {nb_receivers} users` by `You shared "{content_filename}" with {nb_receivers} users`

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 5]
```diff
@@ -18,122 +18,177 @@ msgstr ""
...
 msgid "Deprecated"
 msgstr "Obsolète"
 
-#: tracim_backend/lib/core/content.py:2174
+#: tracim_backend/applications/share/config.py:15
+msgid ""
+"[{website_title}] {emitter_name} shared content {content_filename} with "
+"you"
+msgstr ""
+"[{website_title}] {emitter_name} à partagé le fichier {content_filename} "
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
"[{website_title}] {emitter_name} a partagé le fichier {content_filename} "
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 6]
```diff
@@ -142,33 +197,57 @@ msgstr ""
...
 msgstr "généré par le logiciel tracim"
 
+#: tracim_backend/templates/mail/new_upload_event_body_html.mak:2
+msgid ""
+"{username}(<a href=\"{email_link}\">{email}</a>) shared one or several "
+"files with you in this shared space: <a "
+"href={sharedspace_url}>{sharedspace_name}</a>"
+msgstr ""
+"{username}(<a href=\"{email_link}\">{email}</a>) a partagé un ou "
+"plusieurs fichiersavec vous dans cet espace partagé: <a "
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
"plusieurs fichiers avec vous dans cet espace partagé : <a "
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 7]
```diff
@@ -142,33 +197,57 @@ msgstr ""
...
 msgstr "Bonne Journée :)"
 
-#: tracim_backend/templates/mail/created_account_body_html.mak:18
+#: tracim_backend/templates/mail/created_account_body_html.mak:19
+#: tracim_backend/templates/mail/new_upload_event_body_html.mak:10
 #: tracim_backend/templates/mail/reset_password_body_html.mak:16
+#: tracim_backend/templates/mail/shared_content_to_emitter_body_html.mak:14
+#: tracim_backend/templates/mail/upload_permission_to_emitter_body_html.mak:14
 msgid "Suricat', your digital assistant"
 msgstr "Suricate, Votre assistant numérique"
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
msgstr "Suricate, votre assistant numérique"
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 8]
```diff
@@ -142,33 +197,57 @@ msgstr ""
...
 "note: nous vous recommandons fortement de changer votre mot de passe et "
 "de supprimer cet email dès que vous êtes connecté."
 
-#: tracim_backend/templates/mail/created_account_body_html.mak:17
+#: tracim_backend/templates/mail/created_account_body_html.mak:18
+#: tracim_backend/templates/mail/new_upload_event_body_html.mak:9
+#: tracim_backend/templates/mail/shared_content_to_emitter_body_html.mak:13
+#: tracim_backend/templates/mail/upload_permission_to_emitter_body_html.mak:13
 msgid "Enjoy your day :)"
 msgstr "Bonne Journée :)"
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
msgstr "Bonne journée :)"
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 9]
```diff
@@ -142,33 +197,57 @@ msgstr ""
 "avec vos collaborateurs en vous connectant dans votre espace partagé : "
 "{a_link}"
 
-#: tracim_backend/templates/mail/created_account_body_html.mak:16
+#: tracim_backend/templates/mail/created_account_body_html.mak:17
 msgid ""
 "note: as soon as you are connected, we strongly recommend that you change"
 " your password and delete this email."
 msgstr ""
 "note: nous vous recommandons fortement de changer votre mot de passe et "
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
"note : nous vous recommandons fortement de changer votre mot de passe et "
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po (hunk 7 above)
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
msgstr "Suricate, votre assistant numérique"
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po (hunk 6 above)
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
"plusieurs fichiers avec vous dans cet espace partagé : <a "
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 10]
```diff
@@ -199,3 +278,101 @@ msgstr ""
 "Si le lien ne fonctionne pas, Je vous suggère de copier/coller l'url "
 "entière :"
 
+#: tracim_backend/templates/mail/shared_content_to_emitter_body_html.mak:2
+msgid "You shared the file <i>{content_filename}</i> with:"
+msgstr "Vous avez partagé le fichier  <i>{content_filename}</i> avec :"
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
msgstr "Vous avez partagé le fichier <i>{content_filename}</i> avec :"
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 11]
```diff
@@ -199,3 +278,101 @@ msgstr ""
...
+msgid ""
+"<i><a href=\"{email_link}\">{email}</a></i> at this link: <a "
+"href=\"{url}\">{url}</a>"
+msgstr ""
+"<i><a href=\"{email_link}\">{email}</a></i> via ce lien : <a "
+"href=\"{url}\">{url}</a>"
+
+#: tracim_backend/templates/mail/shared_content_to_emitter_body_html.mak:10
+msgid "This file is protected by this password: \"{share_password}\""
+msgstr "Le fichier est protégé par ce mot de passe: \"{share_password}\""
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
msgstr "Le fichier est protégé par ce mot de passe : \"{share_password}\""
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 12]
```diff
@@ -199,3 +278,101 @@ msgstr ""
...
+"href=\"mailto:{emitter_email}\">{username}</a>) to get the password."
+msgstr ""
+"Le fichier est protégé par un mot de passe, merci de me contacter (<a "
+"href=\"mailto:{emitter_email}\">{username}</a>) pour avoir le mot de "
+"passe."
+
+#: tracim_backend/templates/mail/shared_content_to_receiver_body_html.mak:17
+#: tracim_backend/templates/mail/upload_permission_to_receiver_body_html.mak:13
+msgid "Note: You can also use this link: {url}"
+msgstr "Note: vous pouvez aussi utiliser le lien suivant : {url}"
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
msgstr "Note : vous pouvez aussi utiliser le lien suivant : {url}"
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 13]
```diff
@@ -199,3 +278,101 @@ msgstr ""
...
+
+#: tracim_backend/templates/mail/shared_content_to_receiver_body_html.mak:19
+msgid "Note: You can also use this direct link: {direct_url}"
+msgstr "Note: vous pouvez aussi utiliser le lien direct suivant : {direct_url}"
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
msgstr "Note : vous pouvez aussi utiliser le lien direct suivant : {direct_url}"
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 14]
```diff
@@ -199,3 +278,101 @@ msgstr ""
...
+msgid "{username} (through tracim)"
+msgstr "{username} (via Tracim)"
+
+#: tracim_backend/templates/mail/upload_permission_to_emitter_body_html.mak:2
+msgid ""
+"You allowed the upload in this shared space <a "
+"href={sharedspace_url}>{sharedspace_name}</a> with:"
+msgstr ""
+"Vous avez autorisé le téléversement dans cet espace partagé <a "
+"href={sharedspace_url}>{sharedspace_name}</a> avec:"
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
"href={sharedspace_url}>{sharedspace_name}</a> avec :"
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 15]
```diff
@@ -199,3 +278,101 @@ msgstr ""
...
+msgstr ""
+"Le fichier est protégé par ce mot de passe : "
+"\"{upload_permission_password}\""
+
+#: tracim_backend/templates/mail/upload_permission_to_receiver_body_html.mak:6
+msgid ""
+"I allowed you to upload file in this shared space (number: "
+"{workspace_id}):"
+msgstr ""
+"Je vous a autorisé à téléverser des fichiers dans cet espace "
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
"Je vous ai autorisé à téléverser des fichiers dans cet espace "
```

File: backend/development.ini.sample (hunk 4 above)
Comment (inkhey) [reply backend/development.ini.sample]: done, french translation ?

File: backend/development.ini.sample (hunk 3 above)
Comment (inkhey) [reply backend/development.ini.sample]: done, french translation ?

File: backend/development.ini.sample (hunk 2 above)
Comment (inkhey) [reply backend/development.ini.sample]: done, french translation ?

File: backend/development.ini.sample (hunk 1 above)
Comment (inkhey) [reply backend/development.ini.sample]: done, french translation ?

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po (hunk 15 above)
Comment (inkhey) [reply backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: fixed

File: backend/development.ini.sample (hunk 1 above)
Comment (PhilippeAccorsi) [reply backend/development.ini.sample]: `vous a autorisé à téléverser des fichiers dans l'espace partagé "{workspace_number}"`

File: backend/development.ini.sample (hunk 2 above)
Comment (PhilippeAccorsi) [reply backend/development.ini.sample]: `Vous avez autorisé {nb_receivers} utilisateurs à téléverser des fichiers dans "{workspace_name}"`

File: backend/development.ini.sample (hunk 3 above)
Comment (PhilippeAccorsi) [reply backend/development.ini.sample]: `a partagé le fichier "{content_filename}" avec vous`

File: backend/development.ini.sample (hunk 4 above)
Comment (PhilippeAccorsi) [reply backend/development.ini.sample]: `Vous avez partagé "{content_filename}" avec {nb_receivers} utilisateurs`

File: backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po [hunk 16]
```diff
@@ -14,147 +14,214 @@ msgstr ""
...
+
+#: tracim_backend/applications/upload_permissions/config.py:23
+msgid ""
+"[{website_title}] You allowed {nb_receivers} users to upload files on "
+"\"{workspace_name}\""
+msgstr ""
+
+#: tracim_backend/applications/upload_permissions/config.py:32
+msgid ""
+"[{website_title}] {uploader_username} uploaded {nb_uploaded_contents} "
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po]:
```suggestion
"[{website_title}] {uploader_username} shared {nb_uploaded_contents} files in \"{sharedspace_name}\""
```

File: backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po [hunk 17]
```diff
@@ -14,147 +14,214 @@ msgstr ""
...
+"contents into {workspace_name} share space"
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po]:
```suggestion
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 18]
```diff
@@ -18,122 +18,177 @@ msgstr ""
...
+#: tracim_backend/applications/upload_permissions/config.py:23
+msgid ""
+"[{website_title}] You allowed {nb_receivers} users to upload files on "
+"\"{workspace_name}\""
+msgstr ""
+
+#: tracim_backend/applications/upload_permissions/config.py:32
+msgid ""
+"[{website_title}] {uploader_username} uploaded {nb_uploaded_contents} "
+"contents into {workspace_name} share space"
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 19]
```diff
@@ -18,122 +18,177 @@ msgstr ""
...
+
+#: tracim_backend/applications/upload_permissions/config.py:23
+msgid ""
+"[{website_title}] You allowed {nb_receivers} users to upload files on "
+"\"{workspace_name}\""
+msgstr ""
+
+#: tracim_backend/applications/upload_permissions/config.py:32
+msgid ""
+"[{website_title}] {uploader_username} uploaded {nb_uploaded_contents} "
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
"[{website_title}] {uploader_username} shared {nb_uploaded_contents} files in \"{sharedspace_name}\""
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 20]
```diff
@@ -18,122 +18,177 @@ msgstr ""
...
+"contents into {workspace_name} share space"
+msgstr ""
+"[{website_title}] {uploader_username} a partagé {nb_uploaded_contents} "
+"fichiers dans l'espace partagé {workspace_name}\"\""
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 21]
```diff
@@ -18,122 +18,177 @@ msgstr ""
...
+"[{website_title}] You allowed {nb_receivers} users to upload files on "
+"\"{workspace_name}\""
+msgstr ""
+
+#: tracim_backend/applications/upload_permissions/config.py:32
+msgid ""
+"[{website_title}] {uploader_username} uploaded {nb_uploaded_contents} "
+"contents into {workspace_name} share space"
+msgstr ""
+"[{website_title}] {uploader_username} a partagé {nb_uploaded_contents} "
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
"[{website_title}] {uploader_username} a partagé {nb_uploaded_contents} fichiers dans \"{sharedspace_name}\""
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po (hunk 20 above)
Comment (inkhey) [reply backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: fixed

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po (hunk 19 above)
Comment (inkhey) [reply backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: fixed

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po (hunk 18 above)
Comment (inkhey) [reply backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: fixed

File: backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po (hunk 17 above)
Comment (inkhey) [reply backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po]: fixed

File: backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po (hunk 16 above)
Comment (inkhey) [reply backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po]: fixed

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po (hunk 21 above)
Comment (inkhey) [reply backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: fixed

**Pull request merged**